generate/full.py

```diff
@@ -79,7 +25,7 @@ def main(
     top_k: int = 200,
     temperature: float = 0.8,
     checkpoint_path: Optional[Path] = None,
-    tokenizer_path: Path = Path("checkpoints/lit-llama/tokenizer.model"),
+    tokenizer_path: Path = Path("../checkpoints/lit-llama/tokenizer.model"),
```

Comment:
The assumption from the howtos is that these scripts are called from the repo root, so `..` shouldn't be used